Doc/library/html.parser.rst

```diff
@@ -126,16 +126,16 @@ implementations do nothing (except for :meth:`~HTMLParser.handle_startendtag`):
 
 .. method:: HTMLParser.handle_starttag(tag, attrs)
 
-   This method is called to handle the start of a tag (e.g. ``<div id="main">``).
+   This method is called to handle the start tag of an element (e.g. ``<div id="main">``).
 
    The *tag* argument is the name of the tag converted to lower case. The *attrs*
    argument is a list of ``(name, value)`` pairs containing the attributes found
    inside the tag's ``<>`` brackets.  The *name* will be translated to lower case,
    and quotes in the *value* have been removed, and character and entity references
    have been replaced.
 
-   For instance, for the tag ``<A HREF="https://www.cwi.nl/">``, this method
-   would be called as ``handle_starttag('a', [('href', 'https://www.cwi.nl/')])``.
+   For example, for the tag ``<A HREF="https://www.cwi.nl/">``, the *tag* parameter
+   will be ``'a'`` and the *attrs* parameter will be ``[('href', 'https://www.cwi.nl/')]``.
```

Comment:
Is this gloss necessary?  People reading this page should know that much about python functions.